ipywidgets/static/widgets/js/widget.js

```diff
@@ -524,7 +532,8 @@ define(["nbextensions/widgets/widgets/js/manager",
              * Calls the callback right away is the view is already displayed
              * otherwise, register the callback to the 'displayed' event.
              */
-            if (this.is_displayed) {
+            console.log('`WidgetView.after_displayed` is deprecated. Use the WidgetView.displayed promise instead.');
+            if (this._is_displayed) {
```

Comment:
this should probably just use the Promise now: `this.displayed.then(function() {callback.apply(context)})`